norduser/server.go

```diff
@@ -39,6 +42,13 @@ func (s *Server) Ping(context.Context, *pb.Empty) (*pb.Empty, error) {
 
 func (s *Server) StartFileshare(context.Context, *pb.Empty) (*pb.StartFileshareResponse, error) {
 	log.Println("Starting nordfileshare process")
+
+	fileshareStatus := s.fileshareProcessManager.ProcessStatus()
+	if fileshareStatus == childprocess.Running || fileshareStatus == childprocess.RunningForOtherUser {
+		log.Println("Received start fileshare request but fileshare is already running, status is: ", fileshareStatus)
+		return &pb.StartFileshareResponse{StartFileshareStatus: pb.StartFileshareStatus_SUCCESS}, nil
```

Comment:
If the process is running for another user, is it ok to report success because current user might not be able to use it?

Reply:
Should be enough imo, we start it silently, we do not report anything to the user. At best we could log something on the other side, but we already have norduser logs. Later we can make this more sophisticated if needed.